initial-promises/node-server/promises.cf

```diff
@@ -139,7 +139,7 @@ bundle agent check_disable_agent
 
       # Only define this class when we're ready to die - this is a special class name in "abortclasses"
       "should_not_continue"
-        expression => "should_disable_agent",
+        expression => "should_disable_agent.!run_disabled",
```

Comment:
I don't really get the `run_disabled` name, I'd prefer something like `ignore_disable_agent`. We should also add a comment to explain the reason for this.